### PR TITLE
Issue #6934 - Implement onerror event handler for AbstractWorker

### PR DESCRIPTION
**Pull request by jerneyio**
Please review.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/7445)

### CODE REVIEW

File: components/script/dom/webidls/Worker.webidl [hunk 1]
```diff
@@ -18,4 +18,4 @@ interface Worker : EventTarget {
   void postMessage(any message/*, optional sequence<Transferable> transfer*/);
            attribute EventHandler onmessage;
 };
-Worker implements AbstractWorker;
+Worker implements AbstractWorker;
```
Comment (Manishearth) [inline components/script/dom/webidls/Worker.webidl]: nit: there should be a new line at the end of this file

**Comment by Manishearth** [comment]
Looks good to me with one nit, please amend your second commit with the fix and I'll approve the PR :smile:

**Comment by JoshTheGoldfish** [comment]
Done :)

**Comment by Ms2ger** [comment]
Please rebase on master and squash all commits.

**Pull request merged**